src/installers.ts

```diff
@@ -527,22 +528,41 @@ export const wrapInstall =
     //
     // This is the function that Vortex calls
     async (
-      files: string[],
-      destinationPath: string,
+      filesRelativePaths: string[],
+      destinationDirPath: string,
       _gameId: string,
       _progressDelegate: VortexProgressDelegate,
     ): Promise<VortexInstallResult> => {
     //
       const vortexApi: VortexApi = { ...vortex.api, log: vortexApiThing.log };
 
       vortexApi.log(`info`, `Trying to install using ${installer.type}`);
-      vortexApi.log(`debug`, `Input files:`, files);
+      vortexApi.log(`debug`, `Input files:`, filesRelativePaths);
+
+
+      const treeForInstallers =
+        unwrapTreeIfNecessary(vortexApi, fileTreeFromPaths(filesRelativePaths));
 
-      const treeForInstallers = unwrapTreeIfNecessary(vortexApi, fileTreeFromPaths(files));
       const sourceFileCount = fileCount(treeForInstallers.fileTree);
 
-      const modInfo = modInfoFromArchiveNameOrSynthetic(destinationPath);
-      vortexApi.log(`info`, `Extracted mod info from path: ${destinationPath}`);
+      const archivePathOnDisk = destinationDirPath;
+
+      const archivePath =
+        treeForInstallers.transform !== Transform.Unwrapped
+          ? archivePathOnDisk
+          : pipe(
+            archivePathOnDisk.split(path.sep),
+            filter(not(pathEq(treeForInstallers.wrapperDir))),
+            (parts) => path.join(...parts),
+          );
```

Comment:
For my understanding, we are faking the destinationPath to our installers/modInfo if it was giftwrapped so that it never sees such info as being giftwrapped, but then the path on disk is exactly as it is on the disk, right?

Reply:
Yeah, exactly! I opened an issue to consider just moving the giftwrap info into `ModInfo`. While the 'recursive' unwrapping is kind of neat, it’ll probably be cleaner to store the original tree etc. in `ModInfo` instead of the closure.